src/pynxtools/testing/nexus_conversion.py

```diff
@@ -86,11 +88,15 @@ def convert_to_nexus(
             example_files = self.files_or_dir
         else:
             example_files = sorted(glob(os.path.join(self.files_or_dir, "*")))
-        self.ref_nexus_file = [file for file in example_files if file.endswith(".nxs")][
-            0
-        ]
+
+        if not self.ref_log_file:
+            self.ref_log_file = [
+                file for file in example_files if file.endswith(".log")
+            ][0]
+        assert self.ref_log_file, "Reference nexus .log file not found"
+
         input_files = [file for file in example_files if not file.endswith(".nxs")]
```

Comment:
```suggestion
        input_files = [file for file in example_files if not file.endswith(".log")]
```

Reply:
You are right. It should also probably not end with ".nxs" anyway, so I just implemented a check against both.